### PR TITLE
Feature: Initial beta of implementation of non-existent-namespace error

### PR DESCRIPTION
**Pull request by princefishthrower**
@rollup/plugin-replace added as a dependency, console.warn() implemented as the actual 'error', test added as well. All tests passing!

Resolves https://github.com/rootstrap/react-native-use-styles/issues/1

### CODE REVIEW

**Comment by agustito37** [comment]
Seems great! just tested it and is working as expected. Tell me when it's ready to be merged.

File: tests/core/stylesManager.spec.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import { clearCache, getFromCache } from "../../src/core/globalCache";
 import { GlobalStyles, Styles, GlobalUse } from "../../src/core/stylesManager";
 
+const consoleSpy = jest.spyOn(console, 'warn').mockImplementation()
```
[the remark this answers is not in the document]
Comment (princefishthrower) [reply tests/core/stylesManager.spec.js:4]: @agustito37 - sounds good, I'm on it!

[the remark this answers is not in the document]
Comment (princefishthrower) [reply tests/core/stylesManager.spec.js:4]: @agustito37 - the solution was simpler than I thought. I just needed to add the `--silent` flag to the jest command. Now console logs are hidden from the test output!

[the remark this answers is not in the document]
Comment (agustito37) [reply tests/core/stylesManager.spec.js:4]: I have a concern about that, we might want to know the console output at some moment, maybe when debugging (?). I am not sure if removing the console logs entirely is the best solution here. If this is difficult to implement, or involves a dirty workaround, nevermind, we can live with a log there 😄.

**Comment by princefishthrower** [comment]
@agustito37 - I noticed that the console.warn messages which were still being printed were had the namespace "namespace" in them, not the new one I defined in my tests "not-a-namespace". After some hunting I saw they were actually coming from a different spec, the `globalCache.spec.js` - those are handled now and now there are no console warns printed, and we don't need to use `--silent` 😄

**Comment by agustito37** [comment]
Oh, that's great, yes indeed I mocked a console.warn on some tests. I'll merge this now. Thanks for your contribution!

**Pull request merged**